### ISSUE TITLE
Handle array of entities

### ISSUE THREAD

**Issue by titiyoyo** [theofidry/AliceDataFixtures#206]
I get the following error when trying to import an array of entities (not a collection)

`TypeError: Fidry\AliceDataFixtures\Bridge\Doctrine\Persister\ObjectManagerPersister::getMetadata(): Argument #2 ($object) must be of type object, array given, called in /var/www/vendor/theofidry/alice-data-fixtures/src/Bridge/Doctrine/Persister/ObjectManagerPersister.php on line 158
`

My fixture file
```
App\Entity\EntityA:
  entity_a_1:
    entityBArray: ['@entity_b_1', '@entity_b_2']

App\Entity\EntityB:
  entity_b_{1...2}:
    title: 'test'
```

My mapping info 
```
    <one-to-many field="" target-entity="EntityA" mapped-by="entityBArray" fetch="LAZY">
      <cascade>
        <cascade-persist/>
      </cascade>
    </one-to-many>
```
```
    <one-to-many field="" target-entity="EntityB" mapped-by="entityAArray" fetch="LAZY">
      <cascade>
        <cascade-persist/>
      </cascade>
    </one-to-many>
```

Please check [my PR](https://github.com/theofidry/AliceDataFixtures/pull/207)

PHP 8.1, symfony 5.4, AliceDataFixtures v1.5.1

**Comment by titiyoyo** [comment]
It could if it was merged yes

**Comment by theofidry** [comment]
Closed via #218
